### PR TITLE
Use display table instead of flex in radio control group

### PR DESCRIPTION
**Pull request by mikejolley**
Fixes label wrapping (see https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2425#pullrequestreview-409165842) in radio components.

Before: 
81566459-d2d67600-9392-11ea-8678-b72bbdf07e4f

After:
Screenshot 2020-05-11 at 15 14 45
Screenshot 2020-05-11 at 15 13 54

### CODE REVIEW

File: assets/js/base/components/radio-control/_mixin.scss [hunk 1]
```diff
@@ -24,27 +23,35 @@
...
+		display: table-row;
+
+		span {
+			display: table-cell;
+		}
+
+		.wc-block-radio-control__secondary-label,
+		.wc-block-radio-control__secondary-description {
+			text-align: right;
+			min-width: 50%;
```
Comment (Aljullu) [inline assets/js/base/components/radio-control/_mixin.scss:37]: Would it work removing `min-width` from here? For me, it seems to cause the shipping name to break earlier than necessary:

_with `min-width`:_
imatge

_without `min-width`:_
imatge

Comment (mikejolley) [reply assets/js/base/components/radio-control/_mixin.scss:37]: It's just spans within spans, easy fix :)

**Comment by mikejolley** [comment]
All fixed @Aljullu - just a selector issue.

**Comment by mikejolley** [comment]
@Aljullu I'd prefer a follow up if that is actually an issue. I have a feeling it's there so that when you have secondary labels or descriptions, it forces the secondary one from being pushed too small if the primary description is long.

Thanks!

**Pull request merged**